docs/framework/angular/loader.md

```diff
@@ -132,53 +132,52 @@ If this is a concern, then you can create new base-pages which are tuned to
 a more specific use-case.  For example, suppose we want to create a page
 which *only* has the CiviCase administrative UI.
 
-First, create a skeletal CiviCRM page:
+1. Create a skeletal CiviCRM page:
 
-```
-$ civix generate:page CaseAdmin civicrm/caseadmin
-```
+    ```
+    $ civix generate:page CaseAdmin civicrm/caseadmin
+    ```
 
-Second, edit the new PHP file and update the `run()` function.  Create an
+1. Edit the new PHP file and update the `run()` function.  Create an
 `AngularLoader` to load all the JS/CSS files:
 
-```php
-public function run() {
-  $loader = new \Civi\Angular\AngularLoader();
-  $loader->setModules(array('crmCaseType'));
-  $loader->setPageName('civicrm/caseadmin');
-  $loader->load();
-  parent::run();
-}
-```
-
-Third, initialize AngularJS on the client-side.  You might put this
-in the Smarty template:
-
-```html
-<div ng-app="crmCaseType">
-  <div ng-view=""></div>
-</div>
-```
-
-!!! caution "Security note"
-    When embedding AngularJS within Smarty, it's important that you [do not put untrusted data in Smarty variables](/security/outputs.md#angularjs).
-
-Finally, flush the cache and visit the new page.
-
-```
-# Flush the cache
-$ cv flush
+    ```php
+    public function run() {
+      $loader = new \Civi\Angular\AngularLoader();
+      $loader->setModules(array('crmCaseType'));
+      $loader->setPageName('civicrm/caseadmin');
```

Comment:
When I reviewed https://github.com/civicrm/civicrm-core/pull/10783 locally I found that I needed to call `$loader->useApp()` before calling `$loader->load()`. @totten unless I'm missing something, I think this doc needs to be updated to swap the order of those function calls.